python/paddle/v2/fluid/tests/book/test_fit_a_line.py

```diff
@@ -25,14 +26,17 @@
 feeder = fluid.DataFeeder(place=place, feed_list=[x, y])
 exe = fluid.Executor(place)
 
-exe.run(fluid.default_startup_program())
+exe.run(
+    framework_pb2.ProgramDesc.FromString(fluid.default_startup_program()
```

Comment:
why do serialize and deserialize here?

Reply:
Because this PR changes the `exe.run` to take `ProgramDesc` rather than `ProgramDescBind`.
Anyway, I closed this PR, the reason is here: https://github.com/PaddlePaddle/Paddle/pull/6491#issuecomment-351216105